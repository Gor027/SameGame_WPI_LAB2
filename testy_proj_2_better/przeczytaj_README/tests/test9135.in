pppA A pppApA   pApAAAAppp ppApAp AA A ppApppAAAAp
p AA A p pp Ap pA App   AAAApAp  Ap AAAA    A App  p
 pA ApApp ppAA pA AAp 
p pp AA  p  p   Ap 
 ApAAAAA ApA A pA    ppppAAp AApApAA pppp pA Ap pppp A AA p     AAA  pAAppAA p
AAppA  A  pppppA pAAp pA ApA AA   A pA A p A AAppp A AAppp  p  p   A  ppApp 
 AA
p p AAAp Ap   pp  ppA ppAAp p pA   p    App A pAppp A  AAp pApppApAApA pppApAA  
 AA AA pAA p p  pAAA ApApApp Ap pAAppAApA A  p pp  AA
