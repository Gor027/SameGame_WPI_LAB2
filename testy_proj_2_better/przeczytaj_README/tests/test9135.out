  p
  A
  A  A p
 pp  A p pA     p pA     p    p                 A
 ppAAp p pppAp pA ApA AppAAp Ap   A  A p      AAp
pApAAAAA pppAA pAAAAp ppAAAA AA   pA A Ap p  AAppp p
pAApAAAApApAAA pApAApApApApppAAApAAAApAppApppApppp p   AA              AppA
pApAAAAppApppppAApppppppAApApAppApAppApAAppAAAAApppAAA ppp  p  pAAAA pApAppA p
AAApAAApAApppppppAAAAApApApppApApAAppAApApAAAppppppAAAAAApppApppApAApAppppApAA
